demo/demo.md

```diff
@@ -125,11 +125,11 @@ Use the `pattern` attribute to set custom input validation. This example also us
 Be sure to review this element's API docs for all related information.
 
 <div class="exampleWrapper exampleWrapper--flex">
-  <auro-input id="demo5" pattern="[a-z]{1,15}" required maxlength="15" spellcheck="false" setCustomValidity="Only contain lowercase letters w/no spaces">
+  <auro-input id="demo5" pattern="[a-z]{1,15}" required maxlength="15" spellcheck="false" customValidityBadInput="Only contain lowercase letters w/no spaces">
```

Comment:
This is confusing to me. The scope of the `setCustomValidity` attribute was specifically to allow the user to override the default messaging from the browser. 

The `customValidityBadInput` attribute appears to be bound to a specific string that not coming from the browser, but from this element. 

Also, where is the code that supports `customValidityBadInput`, it doessn't appear to be in this PR? 

Is the change here to deprecate `customValidity` in favor of these invented attributes? 

Also missing from this PR are lines 160-169. These strings shouldn't be in this file, but should be managed by the i18n.js for localization.